index.js

```diff
@@ -132,10 +138,12 @@ function math_block(state, start, end, silent){
             break;
         }
 
-        if(state.src.slice(pos,max).trim().slice(-2)==='$$'){
-            lastPos = state.src.slice(0,max).lastIndexOf('$$');
-            lastLine = state.src.slice(pos,lastPos);
-            found = true;
+        for (const delim of rightDelimiters) {
```

Comment:
is the usage of for-of prohibited for browser compat?

See [the support table](https://caniuse.com/mdn-javascript_statements_for_of)